src/components/MoneyRequestConfirmationList.tsx

```diff
@@ -330,33 +319,6 @@ function MoneyRequestConfirmationList({
     }, [isFocused, transaction, shouldDisplayFieldError, hasSmartScanFailed, didConfirmSplit]);
 
     const routeError = Object.values(transaction?.errorFields?.route ?? {}).at(0);
-
-    useEffect(() => {
-        // We want this effect to run only when the transaction is moving from Self DM to a workspace chat
-        if (!isDistanceRequest || !isMovingTransactionFromTrackExpense || !isPolicyExpenseChat) {
-            return;
-        }
-
-        const errorKey = 'iou.error.invalidRate';
-        const policyRates = DistanceRequestUtils.getMileageRates(policy);
-
-        // If the selected rate belongs to the policy, clear the error
-        if (Object.keys(policyRates).includes(customUnitRateID)) {
-            clearFormErrors([errorKey]);
-            return;
-        }
-
-        // If there is a distance rate in the policy that matches the rate and unit of the currently selected mileage rate, select it automatically
-        const matchingRate = Object.values(policyRates).find((policyRate) => policyRate.rate === mileageRate.rate && policyRate.unit === mileageRate.unit);
-        if (matchingRate?.customUnitRateID) {
-            IOU.setCustomUnitRateID(transactionID, matchingRate.customUnitRateID);
-            return;
-        }
-
-        // If none of the above conditions are met, display the rate error
-        setFormError(errorKey);
-    }, [isDistanceRequest, isPolicyExpenseChat, transactionID, mileageRate, customUnitRateID, policy, isMovingTransactionFromTrackExpense, setFormError, clearFormErrors]);
-
```

Comment:
Conflict was here. We added this useEffect in the PR being reverted [here](https://github.com/Expensify/App/pull/51517/files#diff-29ef0426ea09b0af3420c95911645adcd54c0d3d4c0d8c4fa5aa99dfe37a3646R319-R344). We then CPed [this fix ](https://github.com/Expensify/App/pull/54039), which moved some things around.